test/e2e/config/docker.yaml

```diff
@@ -364,4 +364,4 @@ intervals:
   #  Giving a bit more time during scale tests, we analyze independently if everything works quickly enough.
```

Comment:
Sorry for the back and forth on this file. 

We still need to do these steps:

* Update providers in docker.yaml:
  * Add a new v1.7 entry.
  * ~Remove providers that are not used anymore in clusterctl upgrade tests.~ (no-op because we need to keep v1.5)
  * Change v1.7.99 to v1.8.99.